runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/executor/PythonExecutorLifecycleManagerServiceImpl.java

```diff
@@ -146,9 +146,16 @@ private void doStopPythonExecutor() {
     }
 
     private void doStartPythonExecutor() {
+        PythonExecutorDetails pythonExecutorConfiguration = pythonExecutorConfigurationDataService.getPythonExecutorConfiguration();
         if (!IS_PYTHON_EXECUTOR_EVAL) {
             return;
         }
+        String url = System.getProperty("mgmt.url");
+        String port = url.split(":")[2].split("/")[0];
```

Comment:
```suggestion
        String mgmtUrl = System.getProperty("mgmt.url");
        int port;
        try {
            URL url = new URL(mgmtUrl);
            port = url.getPort();
        } catch (MalformedURLException e) {
            // TODO: handle
        }
```

Reply:
Also extract this lines in a separate method: `boolean isPythonInstalledOnSamePort(PythonExecutorDetails pythonExecutorConfiguration)`